### PR TITLE
Add logic contrastive parameter lambda

### PR DESCRIPTION
**Pull request by Ziyuan-Yang**
Add a logit contrastive parameter lambda for summed contrastive logits. Default to be 0.2.

### CODE REVIEW

**Review by BunsenFeng — CHANGES_REQUESTED** [review]
If you want alpha by default as 0.2, you need to specify it on line 25 like "lambda_ = hyperparameters.get("lambda_", 0.2)" so that even if the user didn't pass in this param this would still get a 0.2. In the current form if the user doesnt pass this param it will throw an error? my understanding

**Comment by Ziyuan-Yang** [comment]
Yeah, I think it is necessary. I will add it.

**Review by BunsenFeng — APPROVED** [review]
lgtm
